src/Courses/TopicGrades/GradeInfoHeader.tsx

```diff
@@ -170,6 +190,9 @@ export const GradeInfoHeader: React.FC<GradeInfoHeaderProps> = ({
 
     return (
         <Grid container spacing={1} style={{ paddingLeft: '1rem' }} alignItems='flex-start'>
+            <Grid item xs={12}>
+                <h3>Topic Score: {topicGrade?.toPercentString() ?? '--'}</h3>
```

Comment:
I feel like this should include the word "total" -- something should distinguish it from the other "scores" on this page...